play/PlayerRules/Basic Rules.txt

```diff
@@ -250,17 +279,17 @@ Species. See chapter 1: Races if you haven't yet.
 
 Leveling up:
 Gain feats according to the table below.
-With Charisma 9 or higher, receive more feats. The feats listed in the table are totals, 
+With Intelligence 9 or higher, receive more feats. The feats listed in the table are totals, 

```

Comment:
Part of the re-balance; something I haven't added yet: charisma granting a unique feat if you put enough points into it.